### ISSUE TITLE
chemical rebalance

### ISSUE THREAD

**Issue by Pnc6lx** [GTNewHorizons/GT-New-Horizons-Modpack#9942]
### Your Pack Version

2.1.2.3

### Your Proposal

chemical balance in gtnh is really bad. for example, you use 3A and 4B make 1C, but you can only get 3A and 4B from electrolysis 7C. i want to change it. 

### Your Goal

rebalance chemical

### Your Vision

more balance chemical, maybe less cost in some chain. 

### Final Checklist

- [X] I have searched this issue tracker and there is nothing similar already. Posting on a closed issue saying *I like this change please reconsider adding it* will prompt us to investigate and reopen it once we confirm your report.
- [X] I understand this change request may not attract enough attention and thus not be implemented.
- [X] I understand this change request may be rejected due to other community members think it's inappropriate.
- [X] I believe this feature would make the pack better.

**Comment by Pnc6lx** [comment]
i will start this work if it is possible

**Comment by Dream-Master** [comment]
Yes chemical is broken. I see this because I change all Aluminium to alumina and broke things even more. Go ahead @Pnc6lx and make suggestions or code prs what we need to change .

**Comment by Elisis** [comment]
This is a major change that would severely alter practically every aspect of the pack's balance. Plus, it would significantly change the feel of the mod.

**Comment by Dream-Master** [comment]
Which is good. But yes let’s discuss this on discord

**Comment by Pxx500** [comment]
Everything is balanced around this mechanic, it should stay this way even if it feels 'odd'. It also forces player to find veins instead of just synthesizing material from scratch. It could also be said that some processes are just not 100% efficient and that's fine imo

Another matter is the fact that it would take weeks if not months to change everything and it would break many things in the process

**Comment by Dream-Master** [comment]
No worries we made developing the pack since 2014. If it take month no worries but this cant be a reason for not changing stuff.

**Comment by Pxx500** [comment]
Imo it's good as it is now, losses are realistic and the fact that you can't just start with anything to get evenything else is quite important

**Comment by Pnc6lx** [comment]
i can remain the loss, but some loss should remove, like electrolysis

**Comment by Dream-Master** [comment]
No i start with some key elements like aluminium and silicon

**Comment by Pxx500** [comment]
> i can remain the loss, but some loss should remove, like electrolysis

Electrolysis loss is a part of the balance, it was meant to be this way

**Comment by Pxx500** [comment]
@Dream-Master can you make a poll in discord to see how many people would like to see this changed?

**Comment by iouter** [comment]
related: https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/issues/7403 ; https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/issues/8975

**Comment by Pnc6lx** [comment]
there is what gtce do [https://github.com/GregTechCEu/gregicality-legacy/pull/388](https://github.com/GregTechCEu/gregicality-legacy/pull/388)
and there is gt5u [https://github.com/Blood-Asp/GT5-Unofficial/pull/1334](https://github.com/Blood-Asp/GT5-Unofficial/pull/1334)
